docs-src/tut-7-rpc.scrbl

```diff
@@ -213,123 +184,42 @@ invoke our @pyin{main} function.
 Now that we have written an entire @|RPS| game in Python it is time to try
 running it.
 
-First we need to generate TLS certificates.
-Think up a pass phrase greater than four characters long, then open a shell in
-your @tt{rps-python} directory and type @tt{./reach tls}:
-
-@verbatim{
-$ ./reach tls
-*** Warning! Certificates generated by `reach tls` are only suitable ***
-    for development purposes and must not be used in production!
-
-Scaffolding auto-generated TLS certificate + key under ./tls directory...
-(When prompted for a pass phrase, please enter between 4 and 1024 characters)
-
-Generating a RSA private key
-....................................................++++
-.................++++
-writing new private key to './tls/reach-server.key'
-Enter PEM pass phrase:
-Verifying - Enter PEM pass phrase:
------
-Signature ok
-subject=CN = reach-server
-Getting Private key
-Enter pass phrase for ./tls/reach-server.key:
-  ...done
-
-When invoking `reach server` you MUST supply the pass phrase you just
-used to generate your certificate as an environment variable, e.g.:
-
-  $ REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
-    REACH_RPC_KEY=YOUR-PRESHARED-KEY \
-    ./reach server
-
-./tls
-├── reach-server.crt
-├── reach-server.csr
-├── reach-server.ext
-└── reach-server.key
-
-0 directories, 4 files
-}
-
-@(hrule)
-
-With those created, now we launch the server:
+First you will need to copy the @tt{index.rsh} file you used for @seclink{tut-7}
+into the directory where you saved @tt{index.py}.
 
+Now open a terminal in that directory and type:
 @verbatim{
-$  REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
-   REACH_RPC_KEY=YOUR-PRESHARED-KEY \
-   ./reach server
-
-===============================
-using rpc key:
-REACH_RPC_KEY=YOUR-PRESHARED-KEY
-===============================
-
-Verifying knowledge assertions
-Verifying for generic connector
-  Verifying when ALL participants are honest
-  Verifying when NO participants are honest
-  Verifying when ONLY "Alice" is honest
-  Verifying when ONLY "Bob" is honest
-Checked 93 theorems; No failures!
-Creating network "rps-python_default" with the default driver
-Creating rps-python_ethereum-devnet_1 ... done
-
-> @"@reach-sh/rpc-server@" app /app
-> node --experimental-modules --unhandled-rejections=strict index.mjs
-
-(node:26) ExperimentalWarning: The ESM module loader is experimental.
+  $ ./reach server
 }
 
-The server is now awaiting connections.
-
-@(hrule)
-
-Open another terminal in your @tt{rps-python} directory and type the following
-to play a game of @|RPS|:
-
 @margin-note{
-Alternatively, if you would prefer to supply configuration as arguments to the
-@pyin{mk_rpc} function at runtime instead of using environment variables, this
-also works:
+Deploying your @|DApp| into production requires obtaining a certificate which
```

Comment:
into production /with the RPC server/

Reply:
Resolved in d05faf748697f53bafeab085dc2aa01625b9ddcf.